### PR TITLE
Change story to activity streams format

### PR DESCRIPTION
**Pull request by maxg0**

### CODE REVIEW

File: stories/models/actor.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from django.utils.translation import ugettext_lazy as _
+from django.db import models
+
+
+class Actor(models.Model):
+    external_id = models.CharField(
+        verbose_name=_('ID'),
```
Comment (tituomin) [inline stories/models/actor.py]: Maybe 'External ID'? or 'Origin ID', this overlaps with the default PK verbose_name

File: aggregaattori/urls.py [hunk 2]
```diff
@@ -27,5 +28,6 @@
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^v1/', include(router.urls)),
+    url(r'^v1/import_activity_stream/', import_activity_stream),
```
Comment (tituomin) [inline aggregaattori/urls.py]: Rename to activity_stream, but I guess you already did...

File: aggregaattori/urls.py [hunk 3]
```diff
@@ -28,6 +28,6 @@
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^v1/', include(router.urls)),
-    url(r'^v1/import_activity_stream/', import_activity_stream),
+    url(r'^v1/activity_stream/', import_activity_streams),
```
Comment (tituomin) [inline aggregaattori/urls.py]: Yes, good.

**Comment by mikkokeskinen** [comment]
I fixed Story model a bit and added tests for the methods.

**Pull request merged**